src/style.css

```diff
@@ -0,0 +1,67 @@
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  background-color: black;
+}
+
+.block {
+  height: 300px;
+  font-size: 100px;
+  color: #fff;
+  font-family: Arial;
+  text-align: center;
+  line-height: 300px;
+  width: 33.33%;
+  min-width: 300px;
+  flex-grow: 1;
+}
+
+.item_1 {
+  background-color: rgb(255, 0, 0);
+  order: 1;
```

Comment:
You don't need to provide order property to those elements, which order should be the same as in HTML.